### DISCUSSION TITLE
What about different styles of visualizations?

### DISCUSSION THREAD

**Discussion by letavocado** [katspaugh/wavesurfer.js#2811] [Ideas]
![image](https://github.com/katspaugh/wavesurfer.js/assets/31704887/2c851db4-c228-499f-afa6-b72472e4ce7b)

Inspired from
https://css-tricks.com/making-an-audio-waveform-visualizer-with-vanilla-javascript/

**Comment by katspaugh** [comment]
Looks cool!
We could perhaps allow passing a custom function for rendering? Perhaps something like `renderPeak(peakIndex, peakValue)`?
